package.json

```diff
@@ -44,10 +44,16 @@
     "monaco-editor": ">=0.30"
   },
   "devDependencies": {
+    "path-browserify": "^1.0.0",
```

Comment:
doesnt work currently when external prepacked - why?
i also noticed that the build `tailwindcss.worker.js` doesnt look like an esm module ...

Reply:
when `path-browserify` and `url` are `dependencies` eg external then one gets this error in the web console:

Uncaught Error: Dynamic require of "path-browserify" is not supported
    at tailwindcss.worker.js:13:11

so in this WIP they are `devDependencies`

Reply:
just removed node_modules and the lock but still the same problem

Reply:
I made a typo. I meand it’s **ok** to bundle dependencies if that’s necessary.

Reply:
👍